dealership/contracts.py

```diff
@@ -1,12 +1,50 @@
-class Contract(object):
-    pass
+from .customers import Customer, Employee
```

Comment:
I'd use concise but very specific/descriptive and frequent commit messages–this makes things easier for the reviewer who can read the commit message and immediately understand what the change is about, and also for yourself when you want to revert to a specific previous commit.

Reply:
will follow that going forwards. 
Thank you so much for reviewing the code your comments are very helpful for me to improve and refine my code.